modules/storage-expanding-csi-volumes.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies
+//
+// * storage/expanding-persistent-volumes.adoc
+
+[id="expanding-csi-volumes_{context}"]
+= Expanding CSI volumes
+
+Having a more advanced storage subsystem is essential to fully supporting critical database requirements. The Container Storage Interface (CSI) allows {product-title} to expand storage volumes after they have already been created.
+
+Support for expanding CSI volumes is enabled by default. However, to use CSI volume expansion, you must have a specific CSI driver that supports volume expansion.
+
+{product-title} does not ship with any CSI drivers. It is recommended to use the CSI drivers provided by
+link:https://kubernetes-csi.github.io/docs/drivers.html[community or storage vendors].
+
+Installation instructions differ by driver, and are found in each driver's documentation. Follow the instructions provided by the CSI driver.
```

Comment:
Not sure if the last sentence here is necessary.

Reply:
Right. I merged this and the preceding paragraph to be more succinct.